Sources/StreamChatUI/CommonViews/TitleContainerView/TitleContainerView.swift

```diff
@@ -10,7 +10,7 @@ open class TitleContainerView: _View, AppearanceProvider, SwiftUIRepresentable {
     /// Content of the view that contains title (first line) and subtitle (second nil)
     open var content: (title: String?, subtitle: String?) = (nil, nil) {
         didSet {
-            updateContentIfNeeded()
+            updateContent()
```

Comment:
Do we need to change this one as well? I think only the update from the delegate is what we need to change 🤔

Reply:
Updated so we don't have to change it here.